setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 setup(name='visionarypm',
-      version='5.0.1',
+      version='5.0.2',
```

Comment:
Nope. We're removing travis because of wheel, so we need to bump the version to allow for proper manual deployment.